src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml

```diff
@@ -0,0 +1,39 @@
+﻿@using EA.Weee.Web.RazorHelpers
+@using MvcNavigationHelpers
+
+@using EA.Weee.Web.Areas.Producer.ViewModels
+
+@model OrganisationDetailsTabsViewModel
+
+@{
+    var @title = "Contact details";
+    ViewBag.Title = @title + " - GOV.UK";
+}
+@section full_main_content{
+    <div class="govuk-!-padding-bottom-8"></div>
+    <div class="govuk-tabs">
+        @Html.Partial("ViewOrganisation/_OrganisationDetailsTabs", Model)
```

Comment:
@seanciv  Sorry should have mentioned and didnt notice that the scheme used javascript. Annoyingly as this is an external screen it will also need to work without javascript so probably best we dont use the javascript method on here.  This one C:\repos\prsd-weee\src\EA.Weee.Web\Areas\Aatf\Controllers\ManageEvidenceNotesController.cs doesnt use javascript I believe it just uses the full views rather than partials.